test/unit/test_config.rb

```diff
@@ -6,52 +6,52 @@
 # Unit test the Config class
 class TestConfig < MiniTest::Test
   # Just copy and paste this in from the README
-  def readme_instructions(t)
+  def readme_instructions(task)
     # Name of quality task.
     # Defaults to :quality.
-    t.quality_name = 'quality'
+    task.quality_name = 'quality'
 
     # Name of ratchet task.
     # Defaults to :ratchet.
-    t.ratchet_name = 'ratchet'
+    task.ratchet_name = 'ratchet'
 
     #
     # Set minimum values to ratchet to.
     #
     # Defaults to { bigfiles: 300 }
     #
-    t.minimum_threshold = { rubocop: 300 }
+    task.minimum_threshold = { rubocop: 300 }
 
     # Array of strings describing tools to be skipped--e.g., ["cane"]
     #
     # Defaults to []
-    t.skip_tools = []
+    task.skip_tools = []
 
     # Log command executation
     #
     # Defaults to false
-    t.verbose = false
+    task.verbose = false
 
     # Relative path to output directory where *_high_water_mark
     # files will be read/written
     #
     # Defaults to 'metrics'
-    t.output_dir = 'metrics'
+    task.output_dir = 'metrics'
 
     # Pipe-separated regexp string describing what to look for in
     # files as 'todo'-like 'punchlist' comments.
     #
     # Defaults to 'XXX|TODO|FIXME|OPTIMIZE|HACK|REVIEW|LATER|FIXIT'
-    t.punchlist_regexp = 'XXX|TODO|FIXME|OPTIMIZE|HACK|REVIEW|LATER|FIXIT'
+    task.punchlist_regexp = 'XXX|TODO|FIXME|OPTIMIZE|HACK|REVIEW|LATER|FIXIT'
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker